src/controller/controldevice/controller/mapping/factories/ButtonMappingFactory.cpp

```diff
@@ -147,74 +153,74 @@ ButtonMappingFactory::CreateDefaultSDLButtonMappings(ShipDeviceIndex shipDeviceI
-                                                                                 SDL_CONTROLLER_AXIS_TRIGGERRIGHT, 1));
+                                                                                 LUS_DEFAULT_CONT_MAPPING_R, 1));
             break;
         case BTN_Z:
             mappings.push_back(std::make_shared<SDLAxisDirectionToButtonMapping>(shipDeviceIndex, portIndex, BTN_Z,
-                                                                                 SDL_CONTROLLER_AXIS_TRIGGERLEFT, 1));
+                                                                                 LUS_DEFAULT_CONT_MAPPING_Z, 1));
             break;
         case BTN_START:
             mappings.push_back(std::make_shared<SDLButtonToButtonMapping>(shipDeviceIndex, portIndex, BTN_START,
-                                                                          SDL_CONTROLLER_BUTTON_START));
+                                                                          LUS_DEFAULT_CONT_MAPPING_START));
             break;
         case BTN_CUP:
             mappings.push_back(std::make_shared<SDLAxisDirectionToButtonMapping>(shipDeviceIndex, portIndex, BTN_CUP,
-                                                                                 SDL_CONTROLLER_AXIS_RIGHTY, -1));
+                                                                                 LUS_DEFAULT_CONT_MAPPING_CUP, -1));
             break;
         case BTN_CDOWN:
             mappings.push_back(std::make_shared<SDLAxisDirectionToButtonMapping>(shipDeviceIndex, portIndex, BTN_CDOWN,
-                                                                                 SDL_CONTROLLER_AXIS_RIGHTY, 1));
+                                                                                 LUS_DEFAULT_CONT_MAPPING_CDOWN, 1));
             if (isGameCube) {
                 mappings.push_back(std::make_shared<SDLButtonToButtonMapping>(shipDeviceIndex, portIndex, BTN_CDOWN,
-                                                                              SDL_CONTROLLER_BUTTON_RIGHTSHOULDER));
+                                                                              LUS_DEFAULT_CONT_MAPPING_CDOWN2));
             }
             break;
         case BTN_CLEFT:
             mappings.push_back(std::make_shared<SDLAxisDirectionToButtonMapping>(shipDeviceIndex, portIndex, BTN_CLEFT,
-                                                                                 SDL_CONTROLLER_AXIS_RIGHTX, -1));
+                                                                                 LUS_DEFAULT_CONT_MAPPING_CLEFT, -1));
             if (isGameCube) {
                 mappings.push_back(std::make_shared<SDLButtonToButtonMapping>(shipDeviceIndex, portIndex, BTN_CLEFT,
-                                                                              SDL_CONTROLLER_BUTTON_Y));
+                                                                              LUS_DEFAULT_CONT_MAPPING_CLEFT2));
```

Comment:
these would benefit from more descriptive names. a consumer of LUS shouldn't have to know that `CLEFT2` is for an `SDLButton` only on gamecube controllers and `CLEFT` is for an `SDLAxisDirection`

Reply:
My bad, I didn't see that it was for gamecube.